examples/gcf-pubsub-vm-delete-event-handler/main.py

```diff
@@ -0,0 +1,660 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+__author__ = ('Jeff McCune <jeff@openinfrastructure.co>, '
+              'Gary Larizza <gary@openinfrastructure.co>')
+
+import base64
+import json
+import logging
+import os
+import google.cloud.logging
+from enum import Enum
+from google.cloud.logging.resource import Resource
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+from typing import List
+
+
+class RuntimeState:
+    """Stores App instance for the lifetime of the process"""
+    pass
+
+
+RuntimeState.app = None
+
+
+class Result(Enum):
+    """The overall result of the DNS cleanup for user reporting
+
+    An OK result indicates the cleanup completed normally.
+
+    A NOT_PROCESSED results is likely a result of losing the race against the
+    VM delete operation and is intended to signal to the user they may need to
+    cleanup DNS records using another mechanism (e.g. manually).
+    """
+    OK = 0
+    NOT_PROCESSED = 1
+
+
+class Severity(Enum):
```

Comment:
This file is rather long, can we move all these logging artifacts [into a package](https://cloud.google.com/functions/docs/writing/#functions-writing-file-structuring-python)? 
- StructuredLog and subclasses
- These Enum subclasses.

Reply:
Sure thing

Reply:
Fixed in f668eb6..20d3bb1